readme.md

```diff
@@ -182,7 +182,7 @@ For advanced JSON usage, check out the [`parseJson`](documentation/2-options.md#
 | Retries on failure    | :heavy_check_mark:  | :x:                | :x:                  | :heavy_check_mark:       | :x:                | :heavy_check_mark:     |
 | Progress events       | :heavy_check_mark:  | :x:                | :x:                  | :heavy_check_mark:\*\*\* | Browser only       | :heavy_check_mark:     |
 | Handles gzip/deflate  | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark:   | :heavy_check_mark:       | :heavy_check_mark: | :heavy_check_mark:     |
-| Advanced timeouts     | :heavy_check_mark:  | :x:                | :x:                  | :x:                      | :x:                | :x:                    |
+| Advanced timeouts     | :heavy_check_mark:  | :x:                | :heavy_check_mark:   | :x:                      | :x:                | :x:                    |
```

Comment:
I can't find any advanced timeouts usage. Can you provide a reference please?

Reply:
https://github.com/node-fetch/node-fetch#request-cancellation-with-abortsignal

Reply:
That's just normal timeout which all HTTP kind support, as far as I can see.

Reply:
I'll close this PR